src/sound/s_sound.h

```diff
@@ -70,6 +70,7 @@ bool S_AreSoundsEquivalent (AActor *actor, int id1, int id2);
 bool S_AreSoundsEquivalent (AActor *actor, const char *name1, const char *name2);
 int S_LookupPlayerSound (const char *playerclass, int gender, const char *logicalname);
 int S_LookupPlayerSound (const char *playerclass, int gender, FSoundID refid);
+const char *GetSoundClass(AActor *pp);
```

Comment:
I would rather move this function to `AActor` class instead of making it "public".
Otherwise, it should be prefixed with `S_` at least.